materialize-firebolt/config.go

```diff
@@ -13,7 +13,7 @@ type config struct {
 	AWSSecretKey string `json:"aws_secret_key,omitempty" jsonschema:"title=AWS Secret Key"`
 	AWSRegion    string `json:"aws_region,omitempty" jsonschema:"title=AWS Region"`
 	S3Bucket     string `json:"s3_bucket" jsonschema:"title=S3 Bucket"`
-	S3Prefix     string `json:"s3_prefix,omitempty" jsonschema:"title=S3 Prefix"`
+	S3Prefix     string `json:"s3_prefix,omitempty" jsonschema:"title=S3 Prefix,default=/"`
```

Comment:
I had to set the default to something other than an empty prefix, because the UI was complaining about this field being empty, even though it is not marked as required.